docs/docs/how-tos/using-vscode.md

```diff
@@ -7,7 +7,7 @@ provides helpful tooling (including debugging) to assist developers in writing
 code. It also has many other functions which non-developers may also find
 useful such a text editing and markdown rendering.
 
-## Getting started
+## Get started
```

Comment:
Was this one intentional? Whichever way is fine with me, just wanted to call it to attention.

Reply:
Getting started has been renamed to get started